xml/security_ldap.xml

```diff
@@ -405,11 +409,80 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
 
   <sect2 xml:id="sec.security.ldap.server.setup">
   <title>Installation and Basic Setup</title>
-   <para>Install the 389-ds package:</para>
+   <para>
+    The <systemitem>389-ds</systemitem> package contains the &ds398; and the
+    administration tools. If the package is not installed yet, install it with
+    the following command:</para>
   <screen>&prompt.sudo;zypper install 389-ds</screen>
- <!--next steps:
-     create instance.inf
-     dscreate from-file etc.-->
+ <procedure>
+  <title>Setting up the &ds389; Instance</title>
+    <para>
+     You create the instance with the <command>dscreate</command> command.
+     It can take a configuration file (<filename>*.inf</filename>) which
+     defines the instance configuration settings. Alternatively, it can be run
+     in an interactive mode. To create a template <filename>*.inf</filename> file,
+     use the <option>create-template</option> option. For more details, see the
+     man page of <command>dscreate</command>.
+    </para>
+    <para>
+     If you want to set up a trial instance, you can use the following
+     example <filename>*.inf</filename> file as a starting point:
+    </para>
+    <example>
+     <title>
+      <filename>Basic Instance Configuration File</filename>
+     </title>
+     <screen># /tmp/istance.inf
+[general]
+config_version = 2
+
+[slapd]
+root_password = <replaceable>YOUR_ADMIN_PASSWORD_HERE</replaceable>
+
+[backend-userroot]
+sample_entries = yes
+suffix = dc=example,dc=com</screen>
+    </example>
+   <para>
+    Before you use this file in the following steps, edit the
+    <parameter>root_password</parameter> parameter. <remark>taroth 2019-02-27:
+    @wbrown: not sure which password needs to be set for 'root_passord' here
+    (I don't think it's the root password of the machine where this instance is
+    running on, or is it?. In other configuration examples, the placeholder is
+    specified as 'YOUR_PASSWORD_FOR_CN=DIRECTORY_MANAGER' - see
+    http://www.port389.org/docs/389ds/howto/howto-install-389.html
+    In short: Which password are we talking about here and what is it used for?
+   </remark> 
+   </para>
+   <step>
+    <para>
+     To create the &ds389; instance from the <filename>/tmp/instance.inf</filename>
+     above, run:
+    </para>
+    <screen>&prompt.sudo;dscreate from-file /tmp/instance.inf</screen>
+    <para>This creates a working LDAP server.</para>
+    <note>
+     <title>Special Option for Docker</title>
```

Comment:
Better to not mention docker at all, since I actually just removed -c, and I think better would be if SUSE just a container to ship. Can we revisit docker once that image is produced?

Reply:
I see, np. Removed the note.